app/models/reservation.rb

```diff
@@ -0,0 +1,26 @@
+class Reservation < ApplicationRecord
+
+  belongs_to :branch
+  belongs_to :customer
+  has_many :line_items, dependent: :destroy, after_add: :update_total
+
+  validates :booking_schedule, presence: true
```

Comment:
what is booking_schedule?

Reply:
booking schedule storing date and time for the reservation.